### PR TITLE
uhh kinda semantic

### PR DESCRIPTION
**Pull request by mazunki**

### CODE REVIEW

File: scifi.d/misc/wwz/index.html [hunk 1]
```diff
@@ -1,202 +1,212 @@
...
+<a href="http://browsehappy.com/">upgrade your browser</a> to improve
+your experience.
+</p>
+<![endif]-->
+    <header>
+      <h1> World war 102: Entire Worlds at War </h1>
+      <i>the war has been going for almost a century! untold billions died! Help! I cannot wake up from this nightmare! I cannot wake up!</i>
+      <i>Do you think Satan or God is afraid of the Old Ones?</i> <br>
+        -a quote from a random soldier in a trench, before getting shot in the head.
+        G
```
Comment (formula-spectre) [inline scifi.d/misc/wwz/index.html:34]: why is there a stray "G"

Comment (mazunki) [reply scifi.d/misc/wwz/index.html:34]: https://www.youtube.com/clip/Ugkxuz1emzc6sTCMifULO8v9mBx_D-Xc9wHc

Comment (mazunki) [reply scifi.d/misc/wwz/index.html:34]: (i have no idea it wasn't intentional)

Comment (formula-spectre) [reply scifi.d/misc/wwz/index.html:34]: YOU DARE USING MY OWN ~~MEMES~~ SPELLS AGAINST ME POTTER?

Comment (mazunki) [reply scifi.d/misc/wwz/index.html:34]: nom nom nom

**Pull request merged**